typing/outcometree.mli

```diff
@@ -58,17 +58,6 @@ type out_value =
   | Oval_variant of string * out_value option
   | Oval_lazy of out_value
 
-<<<<<<< HEAD
-||||||| 121bedcfd2
-type out_type_param = string * (Asttypes.variance * Asttypes.injectivity)
-=======
-type out_type_param = {
-    ot_non_gen: bool;
-    ot_name: string;
-    ot_variance: Asttypes.variance * Asttypes.injectivity
-}
```

Comment:
would be better to take the upstream type here to reduce drift, and get rid of our separate `out_type_param` (moving its extra fields to this type)